integrations/issue_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"net/http"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNoLoginViewIssue(t *testing.T) {
+	assert.NoError(t, models.LoadFixtures())
+
+	req, err := http.NewRequest("GET", "/user1/repo1/issues/1", nil)
```

Comment:
In your test you seem to load `"code.gitea.io/gitea/models".LoadFixtures()` don't know for the other fixture.